reactivesocket-client/src/main/java/io/reactivesocket/client/ClientBuilder.java

```diff
@@ -219,5 +219,15 @@ public void onNext(Collection<T> socketAddresses) {
             null
         );
     }
+
+    @Override
+    public String toString() {
+        return "ClientBuilder("
+            + "source=" + source.toString()
```

Comment:
Even if it is null, string concatenation will not throw `NullPointerException`. Try:

```java
System.out.println("hello" + null);
```